src/api/transaction.rs

```diff
@@ -86,6 +86,65 @@ struct BakerKeysUpdatedMetadata {
     aggregation_key: BakerAggregationVerifyKey,
 }
 
+#[derive(SerdeSerialize)]
+struct BakerSetOpenStatusMetadata {
+    baker_id:    BakerId,
+    open_status: String,
+}
+
+#[derive(SerdeSerialize)]
+struct BakerSetMetadataUrlMetadata {
+    baker_id:     BakerId,
+    metadata_url: String,
+}
+
+#[derive(SerdeSerialize)]
+struct BakerSetTransactionFeeCommissionMetadata {
+    baker_id:                   BakerId,
+    transaction_fee_commission: String,
+}
+
+#[derive(SerdeSerialize)]
+struct BakerSetBakingRewardCommissionMetadata {
+    baker_id:                 BakerId,
+    baking_reward_commission: String,
+}
+
+#[derive(SerdeSerialize)]
+struct BakerSetFinalizationRewardCommissionMetadata {
+    baker_id: BakerId,
+    finalization_reward_commission: String,
+}
+
+#[derive(SerdeSerialize)]
+struct DelegationStakeUpdatedMetadata {
+    delegator_id:   DelegatorId,
+    new_stake_uccd: Amount,
+    increased:      bool,
+}
+
+#[derive(SerdeSerialize)]
+struct DelegationAddedMetadata {
+    delegator_id: DelegatorId,
+}
+
+#[derive(SerdeSerialize)]
+struct DelegationRemovedMetadata {
+    delegator_id: DelegatorId,
+}
+
+#[derive(SerdeSerialize)]
+struct DelegationSetRestakeEarningsMetadata {
+    delegator_id:     DelegatorId,
+    restake_earnings: bool,
+}
+
+#[derive(SerdeSerialize)]
+struct DelegationSetDelegationTargetMetadata {
+    delegator_id:      DelegatorId,
+    delegation_target: String,
+}
```

Comment:
I don't understand why you don't need "Resumed" and "Interrupted" events (for contracts), but you do need these. Is that deliberate, or an omission?

Reply:
Is this where this comment should be? Are you referring to `contract_update_operations`? I think I omitted them because they don't have amounts. But they may of course be relevant to have for other reasons. Will add.

Reply:
And still relevant?

Reply:
It is kind of in the right place. I still don't quite understand why those events are not needed.

Reply:
These structs are just metadata, i.e. extra information that applications/users may care about but rosetta doesn't. Filed https://github.com/Concordium/concordium-rosetta/issues/38 for including it later on.

Reply:
Ok, thanks.